README.md

```diff
@@ -1,16 +1,185 @@
+data in NDS', we can get something that's easier for humans to read **and**
+that the computer can work with.
+
+Provided you understand `Array` literal syntax, your brain loves seeing data in
+this visually "rhythmic" way.  The NDS uses space and patterns to communicate _both_ the
+content inside, as well as describe some  rules (_In this `Array` there are
+`Array`s. It seems like most inner `Array`s have three elements._).
+
+For clarity in thinking and communicating, working with NDS' is a vital skill.
+
+## Nested Data Structures Are Usable by Computers
+
+We said it before, but a computer can't work with a pie chart or a table. The
+closest approximation it can work with is a nested data structure. So NDS' make
+it _easier_ for humans to understand complex data, but they're the _only_ way a
+computer can understand complex data.
+
+## Nested Data Structures Are Transformed to Produce Insight
+
+Early (1940s-1980s) programmers pretty much had one job description:
+
+> "Take a bunch of raw data and output a re-organized version of it, or do a
+> calculation from all of it."
+
+In fact, the complex world of programming was often housed in a department
+called "data processing."
+
+Imagine you're back in that early era of "data processing." Let's now suppose
+someone asks us to:
+
+> Get the element in the `1` index for all the nested data.
+
+Thanks to the human-brain-friendly nested structure, we can look at the code
+and pretty quickly come up with `[3, 4]`. Since we know what the result should
+be thanks to "eyeballing it," coding this solution can be verified quickly:
+
+```ruby
+array_of_arrays = [ # containing Array (of Array)
+  [1, 3, 5], # "Nested" Array 1
+  [2, 4, 6]  # "Nested" Array 2
+]
+
+def find_firsts(src)
+  result = []
+  0.upto(src.count - 1) do |inner_count|
+    result << src[inner_count][1]
+  end
+  result
+end
+
+find_firsts(array_of_arrays) #=> [3, 4]
+```
+
+OK, we admit, determining the second odd or even number greater than zero isn't
+_that_ exciting. But what if each of these inner `Array`s represented three
+estimates, sorted least-to-most on remodeling two different rooms of your
+house? Suddenly this _exact same_ algorithm is much more exciting:
+
+```ruby
+array_of_bid_arrays = [ # containing Array (of Array)
+  [1000, 3000, 5000], # "Nested" Array 1
+  [2000, 4000, 6000]  # "Nested" Array 2
+]
+
+def what_we_can_get_away_with(src)
+  result = 0
+  0.upto(src.count - 1) do |inner_count|
+    result += src[inner_count][0]
+  end
+  result
+end
+
+def if_money_were_no_object(src)
+  result = 0
+  0.upto(src.count - 1) do |inner_count|
+    result += src[inner_count][2]
+  end
+  result
+end
+
+low_high_totals = [
+  what_we_can_get_away_with(array_of_bid_arrays),
+  if_money_were_no_object(array_of_bid_arrays)
+]
+
+p low_high_totals #=> [3000, 11000]
+p low_high_totals[1] - low_high_totals[1] #=> 8000
+```
+
+Wow! A difference of $8,000! The ability to take a nested structure and
```

Comment:
I think this is missing just a bit of explanation. Something like:

In `what_we_can_get_away_with`, we were able to add the the cheapest estimates from both nested arrays, and in `if_money_were_no_object`, the most expensive estimate. Wow! A difference of $8,000!...